platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java

```diff
@@ -300,20 +300,14 @@ public void unsetClickTargetForEventDelegation(IDomNode sourceNode, DomEventType
 	@Override
 	public void scheduleTimeout(IDomTimeoutNode timeoutNode, Double seconds) {
 
-		JS_call("c.scheduleTimeout", timeoutNode.getNodeId(), Math.round(seconds * 1000));
-	}
-
-	@Override
-	public void clearLastScheduledTimeout() {
```

Comment:
I removed `clearLastScheduledTimeout` because it was never called, and it also does not work deterministically. If needed it should be reimplemented.